README.md

```diff
@@ -1,9 +1,32 @@
-## Installation
-- npm install
+# Chat application
 
-## Week 1
+This application is a real-time chat room. People are able to communicate with each other when they are on the same URL at the same time.
 
+![alt text](./screenshots/coverimage.png "Logo Title Text 1")
+
+
+## How to install
+First of all, clone the repo git clone
+
+Install dependencies
+``` npm install```
+
+Run ``` node app.js ```
+(Or fix TODO 1, it will change to `npm start`)
+to start the server on port 3000.
+
+## Features and tooling
+- [x] Server: express
+- [x] Templating: EJS
+- [x] Real-time: Socket.io
+
+## Week 1 progress
 #### Socket.io
-Figuring out socket.io and following the chat tutorial. Refactored the jQuery examples to vanilla JS so that I didn't need to use the library.
+Figuring out socket.io and following the chat tutorial. Refactored the jQuery examples to vanilla JS so that I didn't need to use the jQuery library.
```

Comment:
Used jQuery to much, I'll check the grammar so that this isn't necessary. 
Refactored the jQuery examples to vanilla JS so that I didn't need to use that library.